models/src/it/java/io/sphere/sdk/reviews/ReviewIntegrationTest.java

```diff
@@ -0,0 +1,392 @@
+                        .withAuthorName((String) REVIEWS[0][0]).withTitle((String) REVIEWS[0][1])));
+                assertNotNull(review.getId());
+            });
+            withCustomer(client(), customer -> {
+                Review review = client().execute(new ReviewCreateCommand(ReviewDraft.of(product.getId(), customer.getId())
+                        .withAuthorName((String) REVIEWS[0][0]).withText((String) REVIEWS[0][2])));
+                assertNotNull(review.getId());
+            });
+            withCustomer(client(), customer -> {
+                Review review = client().execute(new ReviewCreateCommand(ReviewDraft.of(product.getId(), customer.getId())
+                        .withAuthorName((String) REVIEWS[0][0]).withScore((Float) REVIEWS[0][3])));
+                assertNotNull(review.getId());
+            });
+            withCustomer(client(), customer -> {
+                Review review = client().execute(new ReviewCreateCommand(ReviewDraft.of(product.getId(), customer.getId())
+                        .withAuthorName((String) REVIEWS[0][0]).withTitle((String) REVIEWS[0][1])
+                        .withText((String) REVIEWS[0][2])));
+                assertNotNull(review.getId());
+            });
+            withCustomer(client(), customer -> {
+                Review review = client().execute(new ReviewCreateCommand(ReviewDraft.of(product.getId(), customer.getId())
+                        .withAuthorName((String) REVIEWS[0][0]).withTitle((String) REVIEWS[0][1])
+                        .withScore((Float) REVIEWS[0][3])));
+                assertNotNull(review.getId());
+            });
+            withCustomer(client(), customer -> {
+                Review review = client().execute(new ReviewCreateCommand(ReviewDraft.of(product.getId(), customer.getId())
+                        .withAuthorName((String) REVIEWS[0][0]).withText((String) REVIEWS[0][2]).withScore((Float) REVIEWS[0][3])));
+                assertNotNull(review.getId());
+            });
+            withCustomer(client(), customer -> {
+                Review review = client().execute(new ReviewCreateCommand(ReviewDraft.of(product.getId(), customer.getId())
+                        .withAuthorName((String) REVIEWS[0][0]).withTitle((String) REVIEWS[0][1])
+                        .withText((String) REVIEWS[0][2]).withScore((Float) REVIEWS[0][3])));
+                assertNotNull(review.getId());
+            });
+        });
+
+    }
+
+    @Test
+    public void testCreate_NoAuthorName() throws Exception {
+        withProduct(client(), product ->
+                withCustomer(client(), customer -> {
+                    Review review = client().execute(new ReviewCreateCommand(ReviewDraft.of(product.getId(), customer.getId())
+                            .withTitle((String) REVIEWS[0][1])));
+                }));
+    }
+
+    @Test
+    public void testGetAll() throws Exception {
```

Comment:
@schleichardt can you please explain why you don't need this test case? Is it covered by something else?